_content/reference/buddhist-monasteries-meditation-centresi-sri-lanka.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Buddhist Forest Monasteries and Meditation Centres in Sri Lanka: Guide for Foreign Buddhist Practitioners"
+external_url: "https://www.lotuslibrary.com/ebooks/buddhist-forest-monasteries-and-meditation-centres-in-sri-lanka/"
+drive_links:
+  - "https://drive.google.com/file/d/1E0M-00of456yw7hEkooE9DHX2yF6edu5/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1NJhtwOziHJP-lYrEzfyXzLuWaxOtIeeb/view?usp=drive_link"
+course: sri-lankan
+tags:
+  - meditation
+year: 2013
```

Comment:
The original version is even older. But was some time after 2008. Maybe put 2010? 🤔